src/components/Carousel.js

```diff
@@ -645,6 +652,7 @@ export default class Carousel extends Component {
           className="BrainhubCarousel__arrows BrainhubCarousel__arrowRight"
           onClick={this.nextSlide}
           ref={el => this.arrowRightNode = el}
+          disabled={value === lastSlideIndex}
```

Comment:
`disabled` prop should be set to true only if `infinite` option is not enabled. Please add checking for it here.